adventure/Ye_olde_store_directory.yaml

```
@@ -3,7 +3,7 @@
 Toadstool Tavern: 11
 Swords'r'us: 14
 Percival's Potions: 9
-Bitchin' Witch Supply Co.: 13
+Bitchin' Witchin' Supply Co.: 13
```

Looks good! Here I am testing a single-line comment as part of a Pull Request review. If I wanted to suggest a change, it would look something like this

```suggestion
Bitchin' Witchin' Supply Co.: 16
```

Let's pretend I want that 13 to be a 16 for some *super important reason*.